reference/api/documents.mdx

```diff
@@ -301,21 +384,67 @@ Delete one document based on its unique id.
 
 You can use this `taskUid` to get more details on [the status of the task](/reference/api/tasks#get-one-task).
 
+## Delete documents by filter
+
+<RouteHighlighter method="POST" route="/indexes/{index_uid}/documents/delete"/>
+
+Delete a selection of documents based on a filter.
+
+### Path parameters
+
+| Name              | Type   | Description                                                            |
+| :---------------- | :----- | :--------------------------------------------------------------------- |
+| **`index_uid`** * | String | [`uid`](/learn/core_concepts/indexes#index-uid) of the requested index |
+
+### Body
+
+A filter expression written as a string or array of array of strings for the documents to be deleted. `filter` will not work without first explicitly adding attributes to the [`filterableAttributes` list](/reference/api/settings#update-filterable-attributes). [Learn more about filters in our dedicated guide.](/learn/advanced/filtering#filter-basics)
```

Comment:
it would be nice if we could put a big emphasis on the fact that **you absolutely need** to make your fields filterable.
Like a warning panel or something.
One user already missed the info, and @brunoocasali was also afraid of it, so I guess it could be good to ensure no one misses it.

Reply:
We could put it in yellow warning boxes 🤔